test/boulder-tools/tag_and_upload.sh

```diff
@@ -2,6 +2,7 @@
 
 set -feuxo pipefail
 
+DIR=$(pwd)
```

Comment:
Please move this variable declaration / initialization to right above where it's used. 👍🏻